### ISSUE TITLE
Deny non-admin requests to futon/fauxton/whatever

### ISSUE THREAD

**Issue by garethbowen** [medic/medic-webapp#4609]
In 2.x we blocked requests to futon in nginx and required users to tunnel in using ssh permissions so it wasn't accessible by non-admin users. As of 3.0 and Docker we no longer use nginx so requests to futon aren't being blocked.

Add `_utils` to the deny list so only couchdb admins can access it. Ensure the URL matching is broad enough to catch slight variations (eg: `//_utils`, `/_utils/nothing`, etc).

Review the 2.x nginx configuration to see if anything else was blocked previously and now isn't.

**Comment by browndav** [comment]
IIRC per our previous product call, we were going to move to denying this at the `medic-api` level rather than depending on an extra hop through nginx (which now that we've solved #5457, would be a pass-through proxy with no added functionality). I think this is architecturally the best place to solve it: it allows us to remove the hop through nginx, will work in both the hosted-by-us ALB scenario and the hosted-by-partner direct to Docker Machine scenario, and is probably a 3-5 line sort of thing.

Does this sound reasonable, and if so, should I patch or do you want to?

**Comment by garethbowen** [comment]
Yes I was thinking this would be in medic-api too. Happy for anyone to pick this up. If you're busy feel free to leave it for a developer.

**Comment by vimemo** [comment]
Blocked by #2734. This might already be covered in that ticket or will need to reuse something from there.

**Comment by garethbowen** [comment]
Duplicate of #2734
